### PR TITLE
Fix variable outline styles

### PR DESCRIPTION
**Pull request by philippfromme**
### Proposed Changes

Adjusts the IBM Carbon styles of the variable outline tab so that it looks more like it's part of the modeler. This is a proposal that needs discussion in terms of how and where to actually fix the IBM Carbon styles and whether the fixes should also work for the next IBM Carbon component we introduce.

#### Before

image

#### After

image

### CODE REVIEW

File: client/src/app/panel/tabs/variable-outline/VariableOutlineTab.less [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    border-left: 1px solid var(--color-grey-225-10-75) !important;
+  }
+
+  .bio-vo-empty-search {
+    align-self: center !important;
+    margin: auto !important;
+    padding: 0 !important;
+  }
+
+  th,
```
Comment (nikku) [inline client/src/app/panel/tabs/variable-outline/VariableOutlineTab.less:58]: This is probably dangerous (due to missing namespacing). 

I'll look into prefixing it.

Comment (philippfromme) [reply client/src/app/panel/tabs/variable-outline/VariableOutlineTab.less:58]: Yes, indeed. Sorry, I marked it as ready to review without reassessing the CSS so it was still very much in a work-in-progress state. 😅

Comment (nikku) [inline client/src/app/panel/tabs/variable-outline/VariableOutlineTab.less:58]:
```suggestion
  .bio-vo-variable-table th,
```

Comment (nikku) [reply client/src/app/panel/tabs/variable-outline/VariableOutlineTab.less:58]: I realized we already apply the whole fixes locally to the variables panel `.bio-vo-tab-content`, so no additional fixes should be needed.

**Comment by nikku** [comment]
Assessment: It fits much better into the existing UX. I was not able to fix the popup issue (https://github.com/camunda/camunda-modeler/issues/4496) but I'll continue to investigate as a follow-up.

Let's merge it for now (as a quick fix), and investigate [further improvements](https://github.com/camunda/camunda-modeler/issues/4511) in the future.

**Pull request merged**